### ISSUE TITLE
Inaccessible Installer

### ISSUE THREAD

**Issue by blindbat84** [openaudible/openaudible#201]
Hello, Open Audible seems to have an inaccessible installer for screen readers. I was unable to read anything in the installer box, or access any buttons that may be present to install it. Closing the app installer required me to kill it in task manager because alt+F4 to close it provided an inaccessible box as well. I hope the app itself is accessible, but I was unable to get that far.

**Comment by openaudible** [comment]
From the command line, the fastest install is: 
.\OpenAudible_win_1.5.3.exe -q -overwrite

This is a duplicate of #183, where you can read the discussion and other options.
